compiler-rt/lib/scudo/standalone/primary64.h

```diff
@@ -155,82 +176,111 @@ template <typename Config> class SizeClassAllocator64 {
 
   void unmapTestOnly() {
     for (uptr I = 0; I < NumClasses; I++) {
-      RegionInfo *Region = getRegionInfo(I);
-      {
-        ScopedLock ML(Region->MMLock);
-        MemMapT MemMap = Region->MemMapInfo.MemMap;
+      auto RegionInfoIter = RegionInfoManager.getRegionInfoIter(I);
```

Comment:
Would this be better as a for loop rather than a do while loop?

Reply:
There must be at least one RegionInfo instance (for multi-regions case, we allocate one in init()) so I take do-while loop when iterating RegionInfos. Let me know if you think it's still better for understanding (I'm fine with either way)

Reply:
Seems reasonable to me.